src/app/components/weight.js

```diff
@@ -1,36 +1,11 @@
-import { range } from './../constants/constants';
+import countWeight from '../base/countWeight';
 
 export default class Weight {
     constructor(container) {
         this.container = container;
-        this.bmi = this.container.querySelector('.health__weight--bmi');
-        this.indexBmi = this.bmi.dataset.bmi;
-        this.category = this.container.querySelector('.js-weightCategory');
-        this.bmi.innerHTML = this.indexBmi;
-        this.countWeight();
-    }
-
-    countWeight = () => {
-        if (this.indexBmi < 15 || this.indexBmi > 25) {
-            this.category.classList.add(range.RANGE_LOW_CLASS);
-            this.bmi.classList.add(range.RANGE_LOW_CLASS);
-            this.overNormal();
-
-        } else if (this.indexBmi > 15 && this.indexBmi < 20) {
-
-            this.category.classList.add(range.RANGE_MIDDLE_CLASS);
-            this.category.innerHTML = 'Close To Normal Weight';
-            this.bmi.classList.add(range.RANGE_MIDDLE_CLASS);
-
-        } else {
-
-            this.category.classList.add(range.RANGE_HEIGHT_CLASS);
-            this.category.innerHTML = 'Normal Weight';
-            this.bmi.classList.add(range.RANGE_HEIGHT_CLASS);
-        }
-    }
+        this.bmi = this.container.querySelector('.js-index-bmi');
+        const indexBmi = this.bmi.dataset.bmi;
```

Comment:
Here you have `bmi` two times in one line. Maybe rename the `data-bmi` to `data-bmi-value` or `this.bmi` to `this.bmiValue`, and `indexBmi` into `bmiIndex` ...it's better when you start your variables with the name and then context.

Take a look at this [naming cheatsheet](https://github.com/kettanaito/naming-cheatsheet#hclc-pattern) made by Artem. There is a naming for functions, but you can apply the same here. You don't have the **action**, but the **high context** is `bmi` and the **low context** is `index` or `value`